test/e2e/scenarios/pod_namespace_selector.go

```diff
@@ -0,0 +1,23 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] Pod and Namespace selector based Network Policies for ingress between clusters", func() {
+	PIt("Should allow communication between selected pods in selected namespace", func() {
+		By("creating listener pod 1 with label 1 in cluster 1 in namespace 1")
+		By("creating connecting pod 2 with label 2 in cluster 2 in namespace 2")
+		By("creating network policy in cluster 1 that allows communication to pod 1 from any pod with label 2 and in namespace 2 in cluster 2")
+		By("testing connectivity between the two pods")
```

Comment:
We still missing another negative test for this, like

creating listener pod 3 with label 1 in cluster 1 in namespace 1
creating connector pod 4 in with label 1 in cluster 2 in namespace 2

Making sure they didn't connect.